src/app/lib/utilities/makeRelativeUrlPath/index.js

```diff
@@ -0,0 +1,15 @@
+const allowList = ['bbc.com', 'bbc.co.uk', 'bbcrussian.com'];
```

Comment:
Can we make this a cosmos or env variable?

Reply:
Probably would be a good approach to have it in Cosmos so that it can be updated fairly easily. I tried adding it locally to the `.env` files, but it has a bit of trouble parsing a string into an array.

We could iterate on this and create a separate issue to put these values in Cosmos if its thats fair?